sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/ParametersCorrectOrderBase.cs

```diff
@@ -67,6 +67,7 @@ public abstract class ParametersCorrectOrderBase<TArgumentSyntax> : SonarDiagnos
                 HasIncorrectlyOrderedParameters(argumentIdentifiers, argumentParameterMappings, parameterNames, identifierNames,
                     analysisContext.SemanticModel))
             {
+                // for VB the symbol does not contain the method syntax reference
```

Comment:
I don't understand the comment.

Reply:
I could leave just the comment in `SonarAnalyzer.UnitTest/TestCases/ParametersCorrectOrder.vb` ?

The idea is that one would expect `secondaryLocations` to be populated for both CS and VB, but in fact it is not. And the VB implementation of `GetMethodDeclarationIdentifierLocation` is called only with null values, so [VisualBasicSyntaxHelper.FindIdentifierLocation](https://sonarcloud.io/code?id=sonaranalyzer-dotnet&line=201&selected=sonaranalyzer-dotnet%3Asrc%2FSonarAnalyzer.VisualBasic%2FHelpers%2FVisualBasicSyntaxHelper.cs) is not called at all

Reply:
Ah! There is probably something off with the structure of my code then...

Reply:
I'd say we can merge this and you can open a ticket if you want to fix the secondary location issue.